src/ts/components/seekbar.ts

```diff
@@ -689,10 +693,6 @@ export class SeekBar extends Component<SeekBarConfig> {
       e.preventDefault();
 
       if (seeking) {
-        // During a seek (when mouse is down or touch move active), we need to stop propagation to avoid
-        // the VR viewport reacting to the moves.
-        e.stopPropagation();
```

Comment:
Why don't you make it conditional here, like above?

Reply:
We have the `mouseTouchHandler` being called which has the conditional in it